clinical_refs/management/commands/seed_refs_diabetes.py

```
@@ -0,0 +1,39 @@
+from django.core.management.base import BaseCommand
+from clinical_refs.models import ClinicalReference
+
+ADA = [
+    {
+        "title": "ADA Standards of Care 2025 – Glycemic Targets",
```

Ruff (0.12.2)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)